tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
     "compilerOptions": {
         "declaration": true,
+        "downlevelIteration": true,
```

Comment:
if we are not supporting IE or Opera mini, we probably don't need this?

Reply:
See lines 218 and 220 of OcTable.tsx

https://stackoverflow.com/questions/33464504/using-spread-syntax-and-new-set-with-typescript

Using `Array.from(...)` we only get 1 key and not the set. Any ideas?

Reply:
Done and fixed the error.